examples/README.md

```diff
@@ -39,7 +39,7 @@ To run the examples using the published version of the package:
         ```bash
         npx cross-env APPCONFIG_CONNECTION_STRING="<appconfig connection string>" 
         ```
-    

```

Comment:
Nit: Changes to this file can be reverted. It only differs in whitespace.

Reply:
reverted by d838076